fix: bundling multiple files specified as CLI arguments (#1717)
Redocly/redocly-cli: docs/commands/bundle.md

## Options

| Option                     | Type     | Description                                                                                                                                                                                                                                                     |
| -------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | apis                       | [string] | List of API description root filenames or names assigned in the `apis` section of your Redocly configuration file. Default values are all names defined in the `apis` section within your configuration file.                                                   |

Reviewer: ```suggestion
| apis                       | [string] | List of API description root filenames or names assigned in the `apis` section of your Redocly configuration file. Default values are names defined in the `apis` section of your configuration file.                                                   |
```

> | --config                   | string   | Specify path to the [configuration file](#use-alternative-configuration-file).                                                                                                                                                                                  |

Reviewer: ```suggestion
| --config                   | string   | Specify the path to the [configuration file](#use-alternative-configuration-file).                                                                                                                                                                                  |
```

| --dereferenced, -d         | boolean  | Generate fully dereferenced bundle.                                                                                                                                                                                                                             |

> | --ext                      | string   | Specify bundled file extension. Possible values are `json`, `yaml`, or `yml`. Default value is `yaml`.                                                                                                                                                          |

Reviewer: ```suggestion
| --ext                      | string   | Specify the bundled file's extension. The possible values are `json`, `yaml`, or `yml`. Default value is `yaml`.                                                                                                                                                          |
```

> | --extends                  | [string] | Can be used in combination with `--lint` to [extend a specific configuration](./lint.md#extend-configuration). Default values are taken from the Redocly configuration file.                                                                                    |

Reviewer: ```suggestion
| --extends                  | [string] | Can be used in combination with `--lint` to [extend a specific configuration](./lint.md#extend-configuration). The default values are taken from the Redocly configuration file.                                                                                    |
```

> | --force, -f                | boolean  | Generate bundle output even when errors occur.                                                                                                                                                                                                                  |

Reviewer: ```suggestion
| --force, -f                | boolean  | Generate a bundle output even when errors occur.                                                                                                                                                                                                                  |
```

| --help                     | boolean  | Show help.                                                                                                                                                                                                                                                      |

> | --keep-url-references, -k  | boolean  | Keep absolute url references.                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| --keep-url-references, -k  | boolean  | Preserves absolute URL references.                                                                                                                                                                                                                                   |
```

| --lint-config              | string   | Specify the severity level for the configuration file. <br/> **Possible values:** `warn`, `error`, `off`. Default value is `warn`.                                                                                                                              |
| --metafile                 | string   | Path for the bundle metadata file.                                                                                                                                                                                                                              |

> | --output, -o               | string   | Name or folder for the bundle file. If you don't specify the file extension, `.yaml` is used by default. If the specified folder doesn't exist, it's created automatically. **If the file specified as the bundler's output already exists, it's overwritten.** |

Reviewer: ```suggestion
| --output, -o               | string   | Name or folder for the bundle file. If you don't specify the file extension, `.yaml` is used by default. If the specified folder doesn't exist, it's created automatically. **Overwrites existing bundler output file.** |
```
